exotic/api/ld.py

```diff
@@ -207,119 +207,13 @@ def set_ld(self, ld0, ld1, ld2, ld3):
-    return
-
-
-if __name__ == "__main__":  # tests
-    stellar_params = {
-        'teff': 6001.0,
-        'teffUncPos': 88.0,
-        'teffUncNeg': -88.0,
-        'met': -0.16,
-        'metUncPos': 0.08,
-        'metUncNeg': -0.08,
-        'logg': 4.22,
-        'loggUncPos': 0.04,
-        'loggUncNeg': -0.04
-    }
-
-    # Test existing filter
-    filter_info1 = {
-        'filter': "CV",
-        'wl_min': None,
-        'wl_max': None,
-        'u0': {"value": None, "uncertainty": None},
-        'u1': {"value": None, "uncertainty": None},
-        'u2': {"value": None, "uncertainty": None},
-        'u3': {"value": None, "uncertainty": None}
-    }
-
-    # Test alias filter
-    filter_info2 = {
-        'filter': "LCO SDSS u'",
-        'wl_min': None,
-        'wl_max': None,
-        'u0': {"value": None, "uncertainty": None},
-        'u1': {"value": None, "uncertainty": None},
-        'u2': {"value": None, "uncertainty": None},
-        'u3': {"value": None, "uncertainty": None}
-    }
-
-    # Test given only FWHM
-    filter_info3 = {
-        'filter': None,
-        'wl_min': "350",
-        'wl_max': "850.0",
-        'u0': {"value": None, "uncertainty": None},
-        'u1': {"value": None, "uncertainty": None},
-        'u2': {"value": None, "uncertainty": None},
-        'u3': {"value": None, "uncertainty": None}
-    }
-
-    # Test custom-entered ld coefficients
-    filter_info4 = {
-        'filter': None,
-        'wl_min': None,
-        'wl_max': None,
-        'u0': {"value": 2.118, "uncertainty": 0.051},
-        'u1': {"value": -3.88, "uncertainty": 0.21},
-        'u2': {"value": 4.39, "uncertainty": 0.27},
-        'u3': {"value": -1.63, "uncertainty": 0.12}
-    }
-
-    filter_info5 = {
-        'filter': 'N/A',
-        'wl_min': 351.2,
-        'wl_max': 3999.,
-        'u0': {"value": 2.118, "uncertainty": 0.051},
-        'u1': {"value": -3.88, "uncertainty": 0.21},
-        'u2': {"value": 4.39, "uncertainty": 0.27},
-        'u3': {"value": -1.63, "uncertainty": 0.12}
-    }
-
-    ld_obj = LimbDarkening(stellar_params)
-    LimbDarkening.standard_list()
-
-    print(ld_obj.check_fwhm(filter_info5))
-    print(ld_obj.check_standard(filter_info5, True, 4))
-    print(ld_obj.check_standard(filter_info5))
-    print(str(filter_info5))
-    test_ld(ld_obj, filter_info5)
-    print(str(filter_info5))
-    print(f"{ld_obj.filter_desc}, {ld_obj.filter_name}, {ld_obj.wl_min}, {ld_obj.wl_max}")
-
-    LimbDarkening.check_fwhm(filter_info5)
-    LimbDarkening.check_fwhm(filter_info3)
-    test_ld(ld_obj, filter_info1)
-    # test_ld(ld_obj, filter_info2)
-    # test_ld(ld_obj, filter_info3)
-    # test_ld(ld_obj, filter_info4)
-
-    ld = [ld_obj.ld0[0], ld_obj.ld1[0], ld_obj.ld2[0], ld_obj.ld3[0]]
-    ld_unc = [ld_obj.ld0[1], ld_obj.ld1[1], ld_obj.ld2[1], ld_obj.ld3[1]]
-    ld_obj.output_ld()
+    def __str__(self):
+        return (f"\nFilter Name: {self.filter_name}"
```

Comment:
In this case, the 'Filter Name' is actually the 'desc' and what's called name is actually 'Filter Abbreviation'... The values should be reversed.

Reply:
I just made the change, and also removed the anachronistic `join` calls.